holoviews/core/spaces.py

```diff
@@ -655,7 +654,7 @@ def reset(self):
         return self
 
 
-    def _cross_product(self, tuple_key, cache):
+    def _cross_product(self, tuple_key, cache, data_slice):
         """
         Returns a new DynamicMap if the key (tuple form) expresses a
         cross product, otherwise returns None. The cache argument is a
```

Comment:
Agreed, docstrings still need updating. I'd love it if there was some way to inherit method docstrings easily, when you override a method.